src/js/common/pages/Campaign/CampaignNewsItemDetailsPage.jsx

```diff
@@ -673,7 +648,7 @@ class CampaignNewsItemDetailsPage extends Component {
                             </BlockedIndicator>
                           </IndicatorButtonWrapper>
                         )}
-                        {(!isSupportersCountMinimumExceeded && !inDraftMode && !inPrivateLabelMode) && (
+                        {(!isSupportersCountMinimumExceeded && !inDraftMode && !inPrivateLabelMode && webAppConfig.ENABLE_PAY_TO_PROMOTE) && (
```

Comment:
This is not related to PAY_TO_PROMOTE and doesn't need to be turned off.

Reply:
Ok!

Reply:
Ok!

Reply:
Line 651